addons/web/static/src/legacy/js/views/list/list_renderer.js

```diff
@@ -983,6 +983,45 @@ var ListRenderer = BasicRenderer.extend({
             $tr.addClass('o_list_no_open');
         }
         this._setDecorationClasses($tr, this.rowDecorations, record);
+        /**
+         * If the record is allowed to be opened and the current view is a list
+         * view (not a list embedded in a form view), add event listeners for
+         * Ctrl+Click and Middle Click to open the record in a new tab.
+         */
+        if (!$tr.hasClass('o_list_no_open')) {
+            $tr.mousedown(function(e){
```

Comment:
What is `e`?  (Not sure if we're going to be following our conventions here or cores?)

```suggestion
            $tr.mousedown(function(e) {
```

Reply:
`e` is an event object, and yeah I was just following core, but I'm not sure what conventions we should be following

Reply:
I figured.  It was more of a question to point out it's non descriptive.  :slightly_smiling_face:

Reply:
Core uses `e` or `ev`. `event` can't be used as a parameter name as it's a global variable on Chrome and can cause issues for users using Chrome, so unfortunately, `e` or `ev` is as descriptive as it gets for event parameters.

Reply:
Oh, is there some sort of limit on what an event can be named?  `event_param` or some other variant isn't acceptable?

Reply:
Fair point, I guess `event_param` could be used.

Reply:
What Core JS Guidelines have to say:

> Do not name a variable event, use ev. This is to avoid bugs on non-Chrome browsers as Chrome is magically assigning a global event variable (so if you use the event variable without declaring it, it will be fine on chrome but crash on every other browser).